experiments/dataSources/worldBank/WorldBankSources.xml

```diff
@@ -0,0 +1,268 @@
+													<filters/>
+													<filtering enableSearch="true" instantSearch="false" instantFilters="true" searchMethod="client" emptySearchBehavior="query" searchPlaceholder="Search"/>
+												</skuid__filterSet>
+											</components>
+										</division>
+									</divisions>
+								</skuid__grid>
+								<skuid__list model="DetailedIndicators" uniqueid="sk-2qKt-30882" pageSize="10">
+									<columns>
+										<column fillRatio="1" widthBehavior="fixed" width="48">
+											<items>
+												<item fieldId="name"/>
+												<item displayType="template" template="{{topics.0.value}}" displayStyle="tertiary"/>
+												<item fieldId="id" displayType="field" displayStyle="secondary"/>
+											</items>
+										</column>
+										<column fillRatio="1">
+											<items>
+												<item fieldId="sourceNote" displayType="field" displayStyle="secondary"/>
+												<item fieldId="sourceOrganization" displayType="field" displayStyle="tertiary"/>
+											<item displayType="template" template="&#10;Data source URL:  /countries/&lt;CountryCode or ALL&gt;/indicators/{{id}}?format=json" displayStyle="secondary"/>
+</items>
+										</column>
+									</columns>
+									<rowActions/>
+									<massActions/>
+									<filtering enableSearch="false" instantSearch="false"/>
+									<interactions>
+										<interaction type="tap">
+											<action type="setCondition" model="SpecificIndicator" condition="indicator" value="{{id}}"/>
+											<action type="requeryModel" model="SpecificIndicator" behavior="standard"/>
+											<action type="emptyModelData">
+												<models>
+													<model>IndicatorName</model>
+												</models>
+											</action>
+											<action type="createRow" model="IndicatorName" appendorprepend="prepend" defaultmodefornewitems="edit" affectedrows="context">
+												<defaults>
+													<default valuesource="fieldvalue" field="Indicator" enclosevalueinquotes="true" value="{{name}}"/>
+													<default valuesource="fieldvalue" field="Source" enclosevalueinquotes="true" value="{{sourceOrganization}}"/>
+													<default valuesource="fieldvalue" field="SourceNote" enclosevalueinquotes="true" value="{{sourceNote}}"/>
+												</defaults>
+											</action>
+											<action type="runComponentAction" componentid="sk-2pOG-6704" action="navigateToStep" stepId="step3"/>
+										</interaction>
+									</interactions>
+								</skuid__list>
+							</components>
+							<buttons>
+								<button label="Back">
+									<actions>
+										<action type="runComponentAction" componentid="sk-2pOG-6704" action="navigateToStep" stepId="step1"/>
+									</actions>
+								</button>
+								<button label="Next" styleSettingsVariant="primary">
+									<actions>
+										<action type="runComponentAction" componentid="sk-2pOG-6704" action="navigateToStep" stepId=""/>
```

Comment:
This next button currently doesn't do anything. But, per the "Disable the step button" comments above, I think we should consider removing these Next steps altogether.
```suggestion
										<action type="runComponentAction" componentid="sk-2pOG-6704" action="navigateToStep" stepId="step3"/>
```

Reply:
@Rnhatch Since we disabled the navigation buttons, are we going to take out these "Next" buttons or no? Or conditionally enable them based on whether or not the data has been queried?